File: DisplaBackend/DisplaBackend/Models/VirtualComprobante.cs
using System;
using System.Collections.Generic;

namespace DisplaBackend.Models
{
    public partial class VirtualComprobante
    {
        public int Id { get; set; }
        public int IdComprobante { get; set; }
        public int IdVentaVirtual { get; set; }
        public decimal CantidadEntregada { get; set; }
    }
}

File: DisplaFrontend/src/app/model/virtualComprobante.ts
import { ComprobanteCliente } from './comprobanteCliente';
import { VentaVirtual } from './ventaVirtual';

export interface VirtualComprobante {
    Id: number,
    CantidadEntregada: number,
    IdComprobante: number,
    IdVentaVirtual: number,

    IdComprobanteNavigation: ComprobanteCliente,
    IdVentaVirtualNavigation: VentaVirtual,
}

